WalletWasabi.Fluent/Models/Wallets/AmountProvider.cs

```diff
@@ -1,19 +1,21 @@
 using NBitcoin;
 using ReactiveUI;
+using System.Reactive.Linq;
+using System.Reactive.Subjects;
 using WalletWasabi.Services;
 
 namespace WalletWasabi.Fluent.Models.Wallets;
 
 [AutoInterface]
 public partial class AmountProvider : ReactiveObject
 {
-	private readonly WasabiSynchronizer _synchronizer;
 	[AutoNotify] private decimal _usdExchangeRate;
 
 	public AmountProvider(WasabiSynchronizer synchronizer)
 	{
-		_synchronizer = synchronizer;
-		BtcToUsdExchangeRates = this.WhenAnyValue(provider => provider._synchronizer.UsdExchangeRate);
+		BtcToUsdExchangeRates = Observable.FromEventPattern<decimal>(synchronizer, nameof(Services.Synchronizer.UsdExchangeRateChanged))
```

Comment:
@SuperJMN can you take a look at this? The wallet displays no USD balance until the second sync event. Do we need to have an initial value?